### PR TITLE
LOG-10817 (minor) Adding render wrapper function

### PR DESCRIPTION
**Pull request by danilosterrapid7**

### CODE REVIEW

**Comment by danilosterrapid7** [comment]
I am going to consume locally based on the git commit as a dependency.

**Review by jgallagher-r7 — APPROVED** [review]

**Comment by danilosterrapid7** [comment]
I still need to update the docs. I will add it soon and I will need a second review on it.

**Comment by danilosterrapid7** [comment]
As I am using the generic table, I won't need it anymore so I am not merging it. Unless people think that this is an interesting change.

**Pull request closed without merging**
